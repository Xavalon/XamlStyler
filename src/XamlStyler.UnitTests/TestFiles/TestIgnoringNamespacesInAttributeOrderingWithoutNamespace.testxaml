<Window x:Class="WpfTests.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:local="clr-namespace:WpfTests"
        mc:Ignorable="d">
    <Grid>
        <TextBlock
			      Text="Hello"
			      DataContext="{x:Null}"
			      Foreground="Black"
			      Background="Black" />
    </Grid>
</Window>
