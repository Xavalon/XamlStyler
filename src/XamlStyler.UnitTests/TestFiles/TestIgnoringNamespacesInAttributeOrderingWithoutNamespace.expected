<Window
    x:Class="WpfTests.MainWindow"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="clr-namespace:WpfTests"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    mc:Ignorable="d">
    <Grid>
        <TextBlock
            Background="Black"
            DataContext="{x:Null}"
            Foreground="Black"
            Text="Hello" />
    </Grid>
</Window>
